libs/store/src/lib/infrastructure/persistence/store/entity-store-server/entity-store.ts

```diff
@@ -0,0 +1,94 @@
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { EntityCollectionService } from '@ngrx/data';
+
+import { CachePolicy, FetchPolicy, IdentityKey } from '../../../../domain/public_api';
+
+//#region Interfaces
+
+export type QueryParams = { [name: string]: string | string[] } | string;
+
+/**
+ * Entity Store interface
+ */
+export interface EntityStore<T> {
+    /**
+     * Query builder reference.
+     * Use it to create a query to underlying entity collection
+     */
+    readonly queryBuilder: any;
+    /**
+     * Get collection of entities
+     */
+    getAll(): Observable<T[]>;
```

Comment:
We should hide getAll, as to fetch more records which probably will require some pagination, you need Query, if this is used by other methods we should not expose this to user

Reply:
removed